spec/useLoadClientScript.spec.tsx

```diff
@@ -34,6 +30,7 @@ describe("useLoadClientScript", () => {
     await new Promise(nostojs)
 
     hook.rerender()
+    await new Promise(resolve => setTimeout(resolve, 1))
```

Comment:
nostojs is executed slightly before script execution finishes, so a small delay is needed